lib/index.js

```diff
@@ -0,0 +1,11 @@
+const config = require('./config')
+let wrapper;
+// Check if nodejs runtime is greater than v8.0.
+// If so return wrapper which can handle async functions.
+if (parseInt(process.version.substring(1,2)) >= 8){
```

Comment:
if an extra dependency and build size are not concerns, this check should be done using the semver package, instead of substring, which will start to fail when node hits v10+.  Otherwise, robustness should be improved.